reporting/sending_metrics_report.md

```diff
@@ -0,0 +1,19 @@
+# Sending Metrics Report
+
```

Comment:
@dbaluku change title to Submit UgandaEMR Analytics Usage Reoport

Reply:
Done